src/html/write_inventory.jl

```diff
@@ -0,0 +1,139 @@
+"""
+Generate the `objects.inv` inventory file.
+
+Write the file `objects.inv` to the root of the HTML build folder, containing an
+inventory of all linkable targets in the documentation (pages, headings, and docstrings).
+
+The `objects.inv` file is compatible with [Sphinx](https://www.sphinx-doc.org/en/master/index.html)
+See [DocInventories](https://juliadocs.org/DocInventories.jl/stable/formats/) for a
+description. The file can be used by [Intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)
+and the [DocumenterInterLinks](https://github.com/JuliaDocs/DocumenterInterLinks.jl/)
+plugin to link into the documentation from other projects.
+"""
+function write_inventory(doc, ctx)
+
+    @info "Writing inventory file."
```

Comment:
This happens unconditionally anyway and is part of the writer, so I don't think we need to print this:

```suggestion
```

Reply:
Sure, sounds good! I'll update that when we reach consensus on the `version` issue and I wrap everything up.

Reply:
Fixed in https://github.com/JuliaDocs/Documenter.jl/pull/2449